ray-operator/controllers/ray/raycluster_controller_unit_test.go

```diff
@@ -1938,7 +1938,7 @@ func TestCalculateStatusWithoutDesiredReplicas(t *testing.T) {
 func TestCalculateStatusWithSuspendedWorkerGroups(t *testing.T) {
 	setupTest(t)
 
-	testRayCluster.Spec.WorkerGroupSpecs[0].Suspend = ptr.To[bool](true)
```

Comment:
maybe open another PR for this?

Reply:
done in https://github.com/ray-project/kuberay/pull/3585